doc/book/src/spec.md

```diff
@@ -1,3 +1,7 @@
+v---- I kind of felt knocked over the head by this section. It seems to be too early in the book. Like, what even is a "job"? Why am I all of the sudden reading Protobuf specs?
```

Comment:
More general feedback...

Reply:
In the recent version I tried to explain that a bit better. I also went to Rust from protobufs. I'm not sure if that helps.